test/suite/test_schema03.py

```diff
@@ -272,9 +272,9 @@ class test_schema03(TieredConfigMixin, wttest.WiredTigerTestCase):
     nindex_scenarios = wtscenario.quick_scenarios('s_index',
         [[1,1,1],[3,2,1],[5,1,3]], [1.0,0.5,1.0])
     idx_args_scenarios = wtscenario.quick_scenarios('s_index_args',
-        ['', ',type=file', ',type=lsm'], [0.5, 0.3, 0.2])
+        ['', ',type=file'], [0.5, 0.5])
     table_args_scenarios = wtscenario.quick_scenarios('s_extra_table_args',
-        ['', ',type=file', ',type=lsm'], [0.5, 0.3, 0.2])
+        ['', ',type=file'], [0.5, 0.5])
```

Comment:
It's not obvious to me if there's a chance the ratio adjustment made here would impact the testing effectiveness. Appreciate any pair of eyes.

Reply:
To me this looks okay